tools/uf2conv.py

```diff
@@ -359,12 +364,31 @@ def error(msg):
         else:
             drives = get_drives()
             if (len(drives) == 0) and (sys.platform == "linux"):
-                rpidisk = glob.glob("/dev/disk/by-id/usb-RPI_RP2*-part1")
-                try:
-                    subprocess.run(["udisksctl", "mount", "--block-device", os.path.realpath(rpidisk[0])])
-                    drives = get_drives()
-                except:
-                    pass # If it fails, no problem since it was a heroic attempt
+                globexpr = "/dev/disk/by-id/usb-RPI_RP2*-part1"
+                rpidisk = glob.glob(globexpr)
+                if len(rpidisk) == 0:
+                    print("Unable to find disk by ID using expression: {}".format(globexpr))
+                else:
+                    try:
+                        cmd = ["udisksctl", "mount", "--block-device", os.path.realpath(rpidisk[0])]
+                        proc_out = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                        if proc_out.returncode == 0:
+                            stdoutput = proc_out.stdout.decode("UTF-8")
+                            match = re.search(r'Mounted\s+.*\s+at\s+([^\.]*)', stdoutput)
```

Comment:
Is this legit on internationalized installs?  I've been bitten before looking for human-readable strings that come from `.po`s...

Reply:
I've reviewed the source for udisckctl and captions aren't localized. Even if the caption were localized, the code would fall back to the previous behavior of using get_drives() to find the new mount point.